services/api/tests/test_app.py

```diff
@@ -91,6 +91,7 @@ def test_get_endpoint(client: TestClient, first_dataset_endpoint: str) -> None:
     assert response.status_code == 422
 
 
+@pytest.mark.skip(reason="no way to test this scenario since there is no endpoint supporting only config")
```

Comment:
I marked this to skip because now that splits supports both config and dataset and if we send missing param for `config`, it wont return error it can reach out to `dataset` . 
See previous config in EndpointConfig:

```
            "/config-names": {InputType.DATASET: ["/config-names"]},
            "/splits": {
                InputType.DATASET: ["/splits"],
                InputType.CONFIG: ["/split-names-from-streaming", "/split-names-from-dataset-info"],
            },
            "/first-rows": {InputType.SPLIT: ["/first-rows"]},
            "/parquet-and-dataset-info": {InputType.DATASET: ["/parquet-and-dataset-info"]},
            "/parquet": {InputType.DATASET: ["/parquet"]},
            "/dataset-info": {InputType.DATASET: ["/dataset-info"]},
            "/sizes": {InputType.DATASET: ["/sizes"]},
```

Reply:
OK. We should keep the test, but create a dedicated processing step, instead of getting it from libcommon. Same for the next test

Reply:
I added other endpoint config specific for the api app testing